### PR TITLE
DOC:Remove supression from ipython directive to make object visible

### PR DESCRIPTION
**Pull request by Winterflower**
Fix for https://github.com/pydata/pandas/issues/11223
@TomAugspurger should these two Ipython blocks be merged into one?

(this is what it looks like at the mo)
pandas-11223

### CODE REVIEW

**Comment by TomAugspurger** [comment]
That'd be fine, or maybe put a note before each saying something like "First some setup" and "with the setup out of the way...". Either way, it'll be an improvement :+1:

**Comment by Winterflower** [comment]
Added some 'verbal sugar' on either side of the Ipython directive blocks. Hope that's ok.

File: doc/source/groupby.rst [hunk 1]
```diff
@@ -268,6 +269,8 @@ natural to group by one of the levels of the hierarchy.
    index = pd.MultiIndex.from_tuples(tuples, names=['first', 'second'])
    s = pd.Series(np.random.randn(8), index=index)
 
+We can then group by one of the levels in ``s``.
+
 .. ipython:: python
 
    s
```
Comment (jorisvandenbossche) [inline doc/source/groupby.rst]: I would put this `s` at the end of the previous block

File: doc/source/groupby.rst [hunk 2]
```diff
@@ -268,6 +269,8 @@ natural to group by one of the levels of the hierarchy.
    index = pd.MultiIndex.from_tuples(tuples, names=['first', 'second'])
```
Comment (jorisvandenbossche) [inline doc/source/groupby.rst]: This is actually a bit a convoluted way to make the multiIndex. The arrays could also be passed directly to `MultiIndex.from_arrays`.

**Comment by jorisvandenbossche** [comment]
Looks good!

I just put some comments, but that is actually more about the current content than your change :-)

**Comment by jorisvandenbossche** [comment]
@Winterflower Do you have some time to update this?

**Comment by Winterflower** [comment]
yep, @jorisvandenbossche will do tonight!

**Comment by jreback** [comment]
@Winterflower ping when pushed & green

**Comment by Winterflower** [comment]
@jreback pushed and green!

**Comment by jreback** [comment]
thanks!

**Pull request merged**